packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -923,4 +881,70 @@ export default class Editor {
     }
 
     //#endregion
+
+    //#region Dark mode APIs
+
+        /**
+     * Set the dark mode state and converts the content.
+     * @param desiredDarkMode The desired status of dark mode. True if the editor should be in dark mode, false if not.

```

Comment:
'desired' is kinda a weird name for this. How do you feel about 'shouldBeDarkMode' or 'nextDarkMode' or something a little more standard?